Makefile.maker.yaml

```
@@ -27,11 +38,12 @@ verbatim: |
   static-check: generate
```

I think we need this also for build-all

```suggestion
  build-all: generate
  static-check: generate
```